### ISSUE TITLE
name not taken in consideration for mysql

### ISSUE THREAD

**Issue by zaher-mh** [emirozer/fake2db#35]
Name argument is not taken in consideration when creating mysql db.
None should replace by args.name here
https://github.com/emirozer/fake2db/blob/master/fake2db/fake2db.py#L161

**Comment by emirozer** [comment]
Thank you for the report, i will take a look at this

**Comment by bhrgu** [comment]
Hi, all! My two cents is that the issue reported above is not specific only for mysql.
Name argument is used only if custom argument is presented, except postgresql.
I mean `if ` statements in fakedb.main():
```
if args.name and args.custom:
    fake_sqlite_handler.fake2db_sqlite_initiator(args.rows,
                                                             args.name, args.custom)
elif args.custom:
    fake_sqlite_handler.fake2db_sqlite_initiator(args.rows, None, args.custom)
else:
    fake_sqlite_handler.fake2db_sqlite_initiator(args.rows)
```
BTW, it looks like a lack of logic, because name argument will be checked later in `database_caller_creator` methods and custom argument will checked in `*initiator` methods.

Maybe we need refactor this behavoir for calling all initiators the way it's implemented for postgresql?
https://github.com/emirozer/fake2db/blob/master/fake2db/fake2db.py#L179

That will make code simpler and avoid excessive conditional checks.

**Comment by emirozer** [comment]
I just released version 0.5.4 to PyPi that contains this fix.
